content/automate/ManageIQ/System/Event/EmsEvent/RHEVM.class/__methods__/update_vm_import_status.rb

```diff
@@ -0,0 +1,13 @@
+#
```

Comment:
@tinaafitz actually here I would most likely appreciate a bit of guidance. Since I appear to be the first adding a method to an Event class in the `System` namespace I could really use an example of how such a test should be structured....